features/bevragen/persoon/adressering/adres-regels/dev/fields-gba.feature

```diff
@@ -0,0 +1,432 @@
+    | burgerservicenummer | 000000310                       |
+    | fields              | adressering.adresregel2         |
+    Dan heeft de response een persoon met de volgende gegevens
+    | naam                                 | waarde        |
+    | gemeenteVanInschrijving.code         | 0518          |
+    | gemeenteVanInschrijving.omschrijving | 's-Gravenhage |
+    En heeft de persoon de volgende 'verblijfplaats' gegevens
+    | naam       | waarde       |
+    | straat     | een straat   |
+    | huisnummer | 1            |
+    | postcode   | 2583XL       |
+    | woonplaats | Scheveningen |
+
+  Scenario: persoon heeft een adres met postcode en zonder woonplaats als verblijfplaats en adresregel2 wordt gevraagd
+    Gegeven de persoon met burgerservicenummer '000000322' heeft de volgende 'verblijfplaats' gegevens
+    | gemeente van inschrijving (09.10) | functie adres (10.10) |
+    | 0518                              | W                     |
+    En de 'verblijfplaats' heeft de volgende 'adres' gegevens
+    | gemeentecode (92.10) | straatnaam (11.10) | huisnummer (11.20) | postcode (11.60) |
+    | 0518                 | een straat         | 1                  | 2584BZ           |
+    Als gba personen wordt gezocht met de volgende parameters
+    | naam                | waarde                          |
+    | type                | RaadpleegMetBurgerservicenummer |
+    | burgerservicenummer | 000000322                       |
+    | fields              | adressering.adresregel2         |
+    Dan heeft de response een persoon met de volgende gegevens
+    | naam                                 | waarde        |
+    | gemeenteVanInschrijving.code         | 0518          |
+    | gemeenteVanInschrijving.omschrijving | 's-Gravenhage |
+    En heeft de persoon de volgende 'verblijfplaats' gegevens
+    | naam       | waarde     |
+    | straat     | een straat |
+    | huisnummer | 1          |
+    | postcode   | 2584BZ     |
+
+  Abstract Scenario: persoon heeft een adres als verblijfplaats en <veld> wordt gevraagd
+    Gegeven de persoon met burgerservicenummer '000000188' heeft de volgende 'verblijfplaats' gegevens
+    | functie adres (10.10) |
+    | W                     |
+    En de 'verblijfplaats' heeft de volgende 'adres' gegevens
+    | gemeentecode (92.10) | straatnaam (11.10)       | huisnummer (11.20) |
+    | 0518                 | Jonkheer van Riemsdijkln | 88                 |
+    Als gba personen wordt gezocht met de volgende parameters
+    | naam                | waarde                          |
+    | type                | RaadpleegMetBurgerservicenummer |
+    | burgerservicenummer | 000000188                       |
+    | fields              | adressering.<veld>              |
+    Dan heeft de response een persoon met alleen de volgende 'verblijfplaats' gegevens
+    | naam       | waarde                   |
+    | straat     | Jonkheer van Riemsdijkln |
```

Comment:
Waarom lever je hier straat? 

Bij adresregel1 is dat nodig im de regel te vullen. 
Bij adresregel2 is dat nodig om te bepalen of de regel geleverd mag worden. 
Voor adresregel3 en land is alleen verblijfplaats buitenland relevant. Dus vraag je om adresregel3 dan is alleen land (ongelijk aan 0000) en regel3 relevant.

Reply:
Dus ik zou hier zetten: Dan heeft de response een persoon met een leeg 'verblijfplaats' object

Reply:
deze laatste opmerking over correctie fields mapping kunnen we oplossen in aparte PR. Issue gemaakt: #1554

Reply:
> Zoiets speelt ook in https://github.com/VNG-Realisatie/Haal-Centraal-BRP-bevragen/blob/develop/features/gba-dev/fields-mapping-gba.csv:
> 
> In adresregels 1, 2 en 3 worden alle 3 de regels gevraagd, in adressering.land alleen land. Bij adresregel3 en land worden straat en huisnummer (m.i. terecht) niet meegevraagd. In de adresregels heb je de regels verblijf buitenland m.i. niet allemaal nodig, met alleen de betreffende regel weet je genoeg. Vraag je bijvoorbeeld adresregel3 en regel1 heeft een waarde, regel3 is leeg, dan lever je geen adresregel3. Vraag je adresregel3 en regel3 heeft wel een waarde, dan lever je wel adresregel3 (wanneer land<>0000)
> 
> Bij land heb je de 3 regels wel nodig, omdat niet alleen land een waarde kan hebben, maar ook ten minste één van de regels een waarde moet hebben.

@CathyDingemanse, @fsamwel, @KayodeBakker , op basis van deze opmerking heb ik scenario's en rules in de fields-gba.feature bestanden aangepast. Willen jullie die nog reviewen? Of kan ik mergen en zien we tzt wel als de implementatie wordt aangepast